common/services/prismic/exhibitions.js

```diff
@@ -64,13 +65,13 @@ function parseExhibitionDoc(document: PrismicDocument): UiExhibition {
   const sizeInKb = Math.round(document.data.textAndCaptionsDocument.size / 1024);
   const textAndCaptionsDocument = isDocumentLink(document.data.textAndCaptionsDocument) ? Object.assign({}, document.data.textAndCaptionsDocument, {sizeInKb}) : null;
   const id = document.id;
-  const format = data.format.slug;
+  const format = data.format.data && asText(data.format.data.title).toLowerCase();
```

Comment:
This still seems like black magic, why not just check on the title in the template?